### ISSUE TITLE
Add a default values for rpc with empty message

### ISSUE THREAD

**Issue by AdrienVannson** [danielgtaylor/python-betterproto#613]
### The Problem

Sometimes, it is needed to declare RPCs that take no parameter. However, it is not possible in gRPC. In such a situation, it is needed to either create an empty message type, or to use `google.protobuf.Empty` ( https://stackoverflow.com/questions/29687243/protobuf-rpc-service-method-without-parameters ).

However, this is not convenient for the final user: an empty message need to be created  and passed to the method each time. It also makes it necessary to add potentially long imports.

### The Ideal Solution

To make this easier, it would be better to add a default value for the parameter containing the request when it is possible. I see two possibilities:
 - Add the default value only for `google.protobug.Empty`
 - Add the default value for all the messages who have no field at all

I think the second solution would be better as it is common to define empty messages in such situations. I can make a PR if you agree, I already have a working version locally.

### The Current Solution

Currently, it is necessary for the user to import by hand `from betterproto.lib.std.google.protobuf import Empty` or the custom message, which is not convenient.

**Comment by Gobot1234** [comment]
Yeah I think this was the behaviour for v1.2.5 and I can agree it's helpful. Might be also useful to perform implicit returns for Empties as well
